liquibase-couchbase/src/test/java/common/matchers/CouchbaseCollectionAssert.java

```diff
@@ -125,4 +127,26 @@ public CouchbaseCollectionAssert hasLockHeldBy(String lockId, String owner) {
 
         return this;
     }
+
+    public CouchbaseCollectionAssert hasIndex(String queryIndexName) {
```

Comment:
`hasQueryIndex()`? And we could simplify parameter name to `name`

Reply:
Checking existence by name is the same for query and primary index. 

But I changed parameter to `name`